service/controller/v2/cloudconfig/template.go

```diff
@@ -647,3 +647,43 @@ ExecStart=-/bin/bash -c "if ! findfs LABEL=$LABEL > /tmp/label.$LABEL; then wipe
 WantedBy=multi-user.target
 `
 )
+
+// TODO: This part should migrate to nginx-ingress-controller helm chart
+// when parametrized charts will be supported.
```

Comment:
Typo on Parameterized but also we (@giantswarm/team-batman) need to get parameters in chart-operator working so this isn't needed! 😄

Reply:
Thanks, fixed :) would love to not have this piece here too!